lib/Common/Memory/RecyclerTelemetryInfo.cpp

```diff
@@ -52,9 +52,28 @@ namespace Memory
         return this->recycler->GetRecyclerID();
     }
 
-    bool RecyclerTelemetryInfo::GetIsConcurrentEnabled() const
+    RecyclerFlagsTableSummary RecyclerTelemetryInfo::GetRecyclerConfigFlags() const
     {
-        return this->recycler->IsConcurrentEnabled();
+        // select set of config flags that we can pack into an int64
+        RecyclerFlagsTableSummary flags;
+        flags.bits = 0;
```

Comment:
Is this the only usage of `bits`? If so, why not remove it and avoid the issues of unions? You could still `memset(flags, 0, sizeof(flags))` if you wanted to zero things out before writing to it.

Reply:
It gets used in ChakraFull to get the uint32 representation for transmission.  There's an internal PR open that corresponds to this.